tutorindigo/templates/indigo/env.config.jsx

```diff
@@ -23,12 +23,46 @@ const AddDarkTheme = () => {
     return options;
   };
 
+  const addDarkThemeToIframes = () => {
+    const iframes = document.getElementsByTagName('iframe');
+    const iframesLength = iframes.length;
+    if (iframesLength > 0) {
+      Array.from({ length: iframesLength }).forEach((_, ind) => {
+        const style = document.createElement('style');
+        style.textContent = `
+          body{
+            background-color: #0D0D0E;
+            color: #ccc;
+          }
+          a {color: #ccc;}
+          a:hover{color: #d3d3d3;}
+          `;
+        if (iframes[ind].contentDocument) { iframes[ind].contentDocument.head.appendChild(style); }
+      });
+    }
+  };
+
   useEffect(() => {
-    const theme =  cookies.get(themeCookie);
+    const theme = cookies.get(themeCookie);
+
+    // When page load, Footer load before than MFE content which says that there is no iframe on page
+    // hence, not append any class. MutationObserver observes changes in DOM and hence appended dark
+    // attributes when iframe added. After 10 sec, we destroy this observer. 
+    // Adding outside dark-theme condition so that we can remove it on Component Unmount
```

Comment:
```suggestion
    // When page loads, Footer loads before MFE content. Since there is no iframe on page,
    // it does not append any class. MutationObserver observes changes in DOM and hence appends dark
    // attributes when iframe is added. After 10 sec, this observer is destroyed to conserve resources. 
    // It has been added outside dark-theme condition so that it can be removed on Component Unmount.
```